src/Console/ModelsCommand.php

```diff
@@ -442,6 +442,11 @@ public function getPropertiesFromTable($model)
 
         foreach ($columns as $column) {
             $name = $column->getName();
+
+            if ($this->hasLowerCaseProperties()) {
```

Comment:
Something got b0rked with your last change, the method `hasLowerCaseProperties` isn't present anymore

Reply:
Re-added the missing function